plugin.go

```diff
@@ -2,7 +2,9 @@ package main
 
```

Comment:
This CI error looks unrelated to my changes.

```
Installing collected packages: setuptools, wheel, pycparser, cffi
86 | Running setup.py install for pycparser: started
87 | Running setup.py install for pycparser: finished with status 'done'
88 | Running setup.py install for cffi: started
89 | Running setup.py install for cffi: finished with status 'error'
90 | ERROR: Command errored out with exit status 1:
91 | command: /usr/bin/python3.7 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-_w8c5tvi/cffi/setup.py'"'"'; __file__='"'"'/tmp/pip-install-_w8c5tvi/cffi/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-mkly6033/install-record.txt --single-version-externally-managed --prefix /tmp/pip-build-env-80v9kp8z/overlay --compile
92 | cwd: /tmp/pip-install-_w8c5tvi/cffi/
93 | Complete output (48 lines):
94 | unable to execute 'gcc': No such file or directory
95 | unable to execute 'gcc': No such file or directory
```
https://cloud.drone.io/drone-plugins/drone-pypi/49/1/4